lntest/itest/lnd_test.go

```diff
@@ -10245,16 +10247,6 @@ func testMultiHopHtlcLocalTimeout(net *lntest.NetworkHarness, t *harnessTest) {
 		t.Fatalf("bob's htlc should have advanced to the second stage: %v", err)
 	}
 
-	// We'll now mine four more blocks. After the 4th block, a transaction
-	// sweeping the HTLC output should be broadcast.
-	if _, err := net.Miner.Node.Generate(4); err != nil {
```

Comment:
I wonder where this 4 block constant came from 🤔

Reply:
4 is the default CSV used throughout our tests.